### ISSUE TITLE
ChangeSignatureCodeRefactoring encountered an Error

### ISSUE THREAD

**Issue by boomalator** [dotnet/roslyn#66086]
Stack says its a null reference. Don't think it's anything I've written.

        I dunno. Doesn't seem too "fixed" to me.
I still get this:
![image](https://user-images.githubusercontent.com/17992669/208830120-4478e24d-2b91-4392-ba7d-4b5aff5c3ba7.png)

Microsoft Visual Studio Professional 2022 (64-bit) - Current
Version 17.4.2

_Originally posted by @boomalator in https://github.com/dotnet/roslyn/issues/44198#issuecomment-1360881859_

**Comment by Youssef1313** [comment]
Can you provide a repro or the stack trace?

**Comment by Youssef1313** [comment]
Note that there are two known `NullReferenceException`s for this feature. One that was fixed in https://github.com/dotnet/roslyn/pull/65307, and one that is tracked by https://github.com/dotnet/roslyn/issues/28292.

Without a stack trace, we can't tell whether you are encountering one of those.

**Comment by boomalator** [comment]
```
System.NullReferenceException : Object reference not set to an instance of an object.
   at Microsoft.CodeAnalysis.ChangeSignature.AbstractChangeSignatureService.GetParameterIndexFromInvocationArgument(SyntaxNode argument,Document document,SemanticModel semanticModel,CancellationToken cancellationToken)
   at async Microsoft.CodeAnalysis.CSharp.ChangeSignature.CSharpChangeSignatureService.GetInvocationSymbolAsync(<Unknown Parameters>)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at async Microsoft.CodeAnalysis.ChangeSignature.AbstractChangeSignatureService.GetChangeSignatureContextAsync(<Unknown Parameters>)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at async Microsoft.CodeAnalysis.ChangeSignature.AbstractChangeSignatureService.GetChangeSignatureCodeActionAsync(<Unknown Parameters>)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at async Microsoft.CodeAnalysis.ChangeSignature.ChangeSignatureCodeRefactoringProvider.ComputeRefactoringsAsync(<Unknown Parameters>)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at async Microsoft.CodeAnalysis.CodeRefactorings.CodeRefactoringService.GetRefactoringFromProviderAsync(<Unknown Parameters>)
```

**Comment by Youssef1313** [comment]
Great. That is the one already fixed in #65307
